File: php/Scripts/tok.py
import requests
import time


def get_tiktok_username(username):
    url = "https://www.tiktok.com/@{username}"
    response = requests.get(url)
    if response.status_code == 200:
        return response.url.split("/")[-1]
    else:
        return None


def follow_user(username):
    url = "https://www.tiktok.com/api/user/follow/?user_id={username}"
    response = requests.post(url)
    if response.status_code == 200:
        return True
    else:
        return False


def main():
    username = "labandicom"
    if not get_tiktok_username(username):
        print("User not found")
        return

    if not follow_user(username):
        print("Failed to follow user")
        return

    print("Successfully followed user")


if __name__ == "__main__":
    main()

File: php/Scripts/tok.php
<?php

// Import the necessary libraries.
require 'vendor/autoload.php';

use TikTok\Api;

// Set the user's TikTok username.
$username = 'example_username';

// Create a new TikTok API instance.
$api = new Api();

// Try to follow the user.
try {
    $api->followUser($username);
    echo 'Successfully followed user';
} catch (Exception $e) {
    echo 'Failed to follow user: ' . $e->getMessage();
}
